indexing-service/src/main/java/io/druid/indexing/common/task/AbstractFixedIntervalTask.java

```diff
@@ -40,6 +40,16 @@ protected AbstractFixedIntervalTask(
 
   protected AbstractFixedIntervalTask(
       String id,
+      TaskResource taskResource,
```

Comment:
there is already a constr, which takes (id,groupId,resource, dataSource,interval)
Can we use that in indexTask, instead of adding a new one ?